### DISCUSSION TITLE
Truth as Disclosedness

### DISCUSSION THREAD

**Discussion by vtomole** [vtomole/vtomole.github.io#45] [General]
Please discuss https://vtomole.com/blog/2023/06/10/truth here.

**Comment by leigholmsted** [comment]
First I have a question - when you talk about determining truth in the "primordial" way versus determining truth in the "derivative" way, does this match up to (ie. correspond... oh the legs on this one lol) later on when you use the terms "correspondence" and "disclosedness"? In other words, are you using the second set of terms in the same way you're using the first set of terms; are they interchangeable?

My question relates to a point I want to make, but I think I can go ahead and make it before hearing the answer (though I may need to revise depending on what you say back) - 

I agree that we all think of and talk about truth as disclosedness. However, what means would humans have to determine truth other than by disclosedness? I see that truth can exist as correspondence, but I don't see how humans could ever come to know truth other than by disclosedness. Thoughts?

**Reply by vtomole** [reply]
> In other words, are you using the second set of terms in the same way you're using the first set of terms; are they interchangeable?

Only the first half is interchangeable: It doesn't matter if you say "primordial truth" or "truth as disclosedness". The "derivative way" is different because although "truth as correspondence" is derivative, there other derivative ways of thinking about truth. For example, the [American Pragmatists](https://en.wikipedia.org/wiki/Pragmatism) believe that truth is "whatever gets us what we want". This is also derivative of primordial truth.

>  I don't see how humans could ever come to know truth other than by disclosedness.

Exactly. We can never get over and above truth as disclosedness because we are mortal. If we lived forever then we would finally get to "Eternal Truth". That's not possible as far as we can tell.

**Comment by leigholmsted** [comment]
> > In other words, are you using the second set of terms in the same way you're using the first set of terms; are they interchangeable?
> 
> Only the first half is interchangeable: It doesn't matter if you say "primordial truth" or "truth as disclosedness". The "derivative way" is different because although "truth as correspondence" is derivative, there other derivative ways of thinking about truth. For example, the [American Pragmatists](https://en.wikipedia.org/wiki/Pragmatism) believe that truth is "whatever gets us what we want". This is also derivative of primordial truth.
> > Gotcha


> > I don't see how humans could ever come to know truth other than by disclosedness.
> 
> Exactly. We can never get over and above truth as disclosedness because we are mortal. If we lived forever then we would finally get to "Eternal Truth". That's not possible as far as we can tell.
> > Why would living forever allow us to know truth as correspondence? The barrier as I see it isn't time, it's that everything we experience is not direct experience, it's experienced through our senses. We have no direct contact with reality. Perhaps if there was a sixth sense that allowed us to directly sense reality, then we would be able to know truth in this way. And perhaps some have suggested this... communicating with God, using intuition...

**Reply by vtomole** [reply]
> It's that everything we experience is not direct experience, it's experienced through our senses. We have no direct contact with reality.

The reason we have no direct contact with reality is because of time. If we lived forever we would be able to try everything that's possible out and keep those that worked.  These would be the eternal truths. Our mortality is the cause of us not having a direct experience of things.
